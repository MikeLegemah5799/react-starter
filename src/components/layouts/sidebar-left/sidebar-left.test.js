import React from 'react';
import renderer from 'react-test-renderer';
import { LayoutSidebarLeft } from '.';

it('renders without crashing', () => {
  const component = renderer.create(<LayoutSidebarLeft />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
