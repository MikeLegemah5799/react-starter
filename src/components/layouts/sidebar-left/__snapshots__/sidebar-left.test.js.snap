// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<aside
  id="app-layout--sidebar_left"
/>
`;
